features/all-positions/AllPositions.tsx

```diff
@@ -46,20 +53,28 @@ const AllPositions = () => {
     return utils.commify(x as string);
   };
 
+  const prettyAddress = (x: String | null) => {
+    if (!x) return "N/A";
+    return x.substr(0, 6) + "..." + x.substr(x.length - 6, x.length);
+  };
+
   const getCollateralRatio = (
     collateral: BigNumberish,
     tokens: BigNumberish
   ) => {
     if (!latestPrice) return null;
     const tokensScaled = Number(tokens) * Number(latestPrice);
-    return Number(collateral) / tokensScaled;
+    return (Number(collateral) / tokensScaled).toFixed(4);
```

Comment:
I think it's worth moving the `.toFixed(4)` into a `prettyRatio()` method because we might want to also use it on the LockedCollateral and TokensOutstanding figures. WDYT?

Reply:
could do that. For now I think it's better to leave the `toFixed` within the given functions as it keeps the logic in the HTML cleaner. I added `toFixed` in the `prettyBalance` and now the table looks like this:
![image](https://user-images.githubusercontent.com/12886084/88047101-5fe73980-cb51-11ea-8c18-40eaec18d850.png)

Which I think is ideal.

Reply:
>For now I think it's better to leave the toFixed within the given functions as it keeps the logic in the HTML cleaner. I added toFixed in the prettyBalance and now the table looks like this:

Sounds good